include/deal.II/dofs/dof_accessor.h

```diff
@@ -1743,6 +1743,10 @@ class DoFCellAccessor : public DoFAccessor<dimension_,
    * of the respective finite element class for a description of what the
    * prolongation matrices represent in this case.
    *
+   * @note Cells set the values of DoFs independently and might override set
+   * values. By setting @p perform_check, you can enable a check that the
+   * values are in the same ballpark.
```

Comment:
```suggestion
   * values in the global vector, for example when calling the same function earlier
   * from a different cell. By setting @p perform_check, you can enable a check that the
   * previous value and the one to be set here are at least roughly the same. In practice,
   * the may be slightly different because they are computed in a way that theoretically
   * ensures that they are the same, but in practice they are only equal up to round-off.
```